### PR TITLE
Fix typos in README.md

### PR DESCRIPTION
**Pull request by arpitjain099**
- There was a lagging comma instead of a dot
- Some other minor changes

### CODE REVIEW

**Comment by arpitjain099** [comment]
Request review from @mxyng @dhiltgen (can't add to reviewer list on the PR)

File: README.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ I'm a basic program that prints the famous "Hello, world!" message to the consol
 The image features a yellow smiley face, which is likely the central focus of the picture.
 ```
 
-### Pass in prompt as arguments
```
Comment (bmizerany) [inline README.md:176]:
```suggestion
### Pass in prompt as an argument
```

File: README.md [hunk 2]
```diff
@@ -173,7 +173,7 @@ I'm a basic program that prints the famous "Hello, world!" message to the consol
...
+### Pass in the prompt as arguments
```
Comment (bmizerany) [inline README.md:176]:
```suggestion
### Pass the prompt as an argument
```

**Comment by bmizerany** [comment]
Merged. Than you!

**Pull request merged**